src/nlp/problems/hs14.jl

```diff
@@ -136,6 +172,22 @@ function NLPModels.jtprod!(nlp::HS14, x::AbstractVector, v::AbstractVector, Jtv:
   return Jtv
 end
 
+function NLPModels.jtprod_lin!(nlp::HS14, x::AbstractVector, v::AbstractVector, Jtv::AbstractVector)
```

Comment:
You forgot to remove `jtprod` above, so the lines below are missed.

Reply:
It was done on purpose, but you are right. I should probably remove that. Thanks

Reply:
In that case you remove the `jprod` but not `jtprod`, so one of them would need to be changed.
To keep the general function and also the `lin` and `nln` versions, we need to implement the linear API tests and have them optional but off by default.